x/keysigner/keystoresigner.go

```diff
@@ -77,7 +77,15 @@ func (kss *KeystoreSigner) String() string {
 }
 
 func (kss *KeystoreSigner) GetAuth(chainID *big.Int) (*bind.TransactOpts, error) {
-	return bind.NewKeyStoreTransactorWithChainID(kss.keystore, kss.account, chainID)
+	opts, err := bind.NewKeyStoreTransactorWithChainID(kss.keystore, kss.account, chainID)
+	if err != nil {
+		return nil, err
+	}
+
+	opts.GasLimit = 1_000_000
```

Comment:
Wouldn’t it be better to declare those variables as constants or read them from the config?